lib/src/entities/message.dart

```diff
@@ -71,6 +71,6 @@ class Message {
   String html;
   List<Attachment> attachments = [];
 
-  Iterable<Address> _asAddresses(Iterable<dynamic> as) =>
-      (as ?? []).map((a) => a is String ? Address(a) : a as Address);
+  static Iterable<Address> _asAddresses(Iterable<dynamic> adrs) =>
+      adrs == null ? []: adrs.map((a) => a is String ? Address(a) : a as Address);
```

Comment:
Is this in some way better?
(I don't mind.  Honest question)

Reply:
Technically, the performance shall be a little better since no need to pass `this` to the function.